Header header

# 5G Sierra AT KPI

string net_type
float64 rsrp
float64 rsrq
float64 sinr
string band
int16 bw_dl
int16 bw_ul
int16 cell_id
int16 gNB
int16 sector_ID
int16 mimo_dl
int16 mimo_ul
float64 tx_power
float64 rssi_0
float64 rssi_1
float64 rssi_2
float64 rssi_3
